hoot-core-test/src/test/cpp/hoot/core/algorithms/changeset/TestOsmChangesetProvider.h

```diff
@@ -80,21 +80,38 @@ class TestOsmChangesetProvider : public ChangesetProvider
     Change change;
     Change::ChangeType changeType = _getRandomType();
 
+    long version = 0;
+    // Changeset writers verify that modified/deleted features don't have a starting (zero) version.
+    if (changeType != Change::ChangeType::Create)
+    {
+      version = 1;
+    }
+
```

Comment:
Changesets with modify/delete commands and version=0 were giving me quite a bit of grief, so added this check.